t/fake/tests/context.pm

```diff
@@ -0,0 +1,11 @@
+use 5.018;
+use base 'basetest';
+
+sub run {
+    my ($self) = @_;
```

Comment:
Wouldn't be practical if we can access to the context either by:

```perl 
sub run {
     my ($self, $ctx) = @_;
     ....
}
```

or, if inside the test object:


```perl 
sub run {
     my ($self) = @_;
     $ctx = $self->ctx(); # or also with an alias to ->context();
}
```

Reply:
I am happy to go with whatever people want.